collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -299,6 +304,14 @@ public synchronized Tracker getDefaultTracker() {
         return tracker;
     }
 
+    public void logEvent(String event, Bundle bundle) {
```

Comment:
How about calling this something like `logRemoteAnalytics`? There are so many different logs that it's hard to keep track of them all. 

I'd also recommend passing in the action and label strings. That way you can currently include both Firebase Analytics and Google Analytics implementations here and simply remove Google Analytics here either as part of this change or in the future. That way you also don't need to add new constants because the literal strings will only be used once.

Reply:
Thanks for the review. I've updated the pr.